pkg/debugapi/postage.go

```diff
@@ -7,9 +7,24 @@ package debugapi
 import (
 	"net/http"
 
+	"github.com/ethersphere/bee/pkg/bigint"
 	"github.com/ethersphere/bee/pkg/jsonhttp"
 )
 
+type reserveStateResponse struct {
```

Comment:
since we already have two types for reserve state maybe it would be better to reuse the one at `pkg/postage/reservestate.go` which was created just for the api response? or remove that one and use this one?

Reply:
Well, that depends on how the `pkg/postage/reservestate.go` is used, which I can't really assess. I was following the style of other API endpoints that have their own response types. What I am not sure is if this `pkg/postage/reservestate.go`'s type is used also internally for something else then API return type. Because I believe I would have to change the `big.Int` into this new `bigint.BigInt` type which is not meant for internal computation. 
So what do you think?